visivo/commands/serve_phase.py

```diff
@@ -36,13 +41,14 @@ def app_phase(output_dir, working_dir, default_source, name_filter, threads):
     )
     app.config["SEND_FILE_MAX_AGE_DEFAULT"] = 0
 
-    run_phase(
+    runner = run_phase(
         output_dir=output_dir,
         working_dir=working_dir,
         default_source=default_source,
         name_filter=name_filter,
         threads=threads,
     )
+    write_dag(project=runner.project, output_dir=output_dir)
```

Comment:
@tgsoverly are you planning on adding this to run and deploy in a separate PR so we can show it in the app?

Reply:
Yes, I would like to try it in serve for a bit first